sematic/api/endpoints/request_parameters.py

```diff
@@ -25,25 +32,44 @@
 
 
 def get_request_parameters(
-    args: Dict[str, str], model: type
+    args: Dict[str, str],
+    model: type,
+    default_order: Literal["asc", "desc"] = "desc",
 ) -> Tuple[
-    int, Optional[str], Optional[sqlalchemy.Column], Optional[BooleanClauseList]
+    int,
+    Callable,
+    Optional[str],
+    Optional[sqlalchemy.Column],
+    Optional[BooleanClauseList],
 ]:
     """
     Extract, validate, and format query parameters.
 
     Parameters
     ----------
     args : Dict[str, str]
-        request argument as returned by `flask.request.args`.
+        The request argument as returned by `flask.request.args`.
```

Comment:
`request` is available as a [context local](https://flask.palletsprojects.com/en/2.2.x/reqcontext/) even if not passing inn as a parameter.